### PR TITLE
docs:Auto-setter docs

### PR DESCRIPTION
**Pull request by phanimarupaka**
@frankfarzan @mortent 

Kpt offers gcloud auto-setters which are auto-set when `kpt pkg get` is invoked.

### CODE REVIEW

File: site/content/en/guides/producer/setters/_index.md [hunk 1]
```diff
@@ -87,6 +87,19 @@ spec:
   replicas: 3 # {"$kpt-set":"replicas"}
 ```
 
+#### Auto setters
+
+kpt auto-sets gcloud config setters when the package is fetched using `kpt pkg get`.
```
[the remark this answers is not in the document]
Comment (frankfarzan) [reply site/content/en/guides/producer/setters/_index.md:92]: +1. Also why is this in the producer guide? Doesn't the consumer need to understand this mechanism?

Comment (frankfarzan) [inline site/content/en/guides/producer/setters/_index.md:92]: Rephrase "kpt auto-sets gcloud config setters when the package is fetched using `kpt pkg get`."

It doesn't' really parse. Something along the lines of:

Some setters are automatically set when ...

[the remark this answers is not in the document]
Comment (phanimarupaka) [reply site/content/en/guides/producer/setters/_index.md:92]: Yes consumer need not understand it as they get auto populated when they `kpt get` the package.

Comment (phanimarupaka) [reply site/content/en/guides/producer/setters/_index.md]: @frankfarzan Done.

**Review by frankfarzan — APPROVED** [review]

**Pull request merged**